docs/source/demos/cost_benefit_matrix.ipynb

```diff
@@ -122,7 +122,8 @@
    "outputs": [],
    "source": [
     "from evalml import AutoMLSearch\n",
-    "automl = AutoMLSearch(X_train=X_train, y_train=y_train, problem_type='binary', objective='log loss binary')\n",
+    "automl = AutoMLSearch(X_train=X_train, y_train=y_train, problem_type='binary', objective='log loss binary',\n",
+    "                      max_iterations=5)\n",
```

Comment:
Seems fine. We'll probably have to revisit this when the new automl algo comes in because `max_iterations` may no longer be supported at that point.

Reply:
Agreed! Hopefully there's a "super fast" mode hehe